platform/platform-sportpaper/src/main/java/tc/oc/pgm/platform/sportpaper/SpMiscUtil.java

```diff
@@ -83,4 +89,19 @@ public ThrownPotion spawnPotion(Location loc, ItemStack item) {
   public double getArrowDamage(Arrow arrow) {
     return arrow.spigot().getDamage();
   }
+
+  @Override
+  public int getWorldDataVersion(Path levelDatPath) {
+    try {
+      var compoundTag =
+          NBTCompressedStreamTools.a(Files.newInputStream(levelDatPath, StandardOpenOption.READ));
```

Comment:
style: keep the root tag name consistent with modern
```suggestion
      var root =
          NBTCompressedStreamTools.a(Files.newInputStream(levelDatPath, StandardOpenOption.READ));
```